testapps/ApplicationWithParseErrors/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1 @@
+﻿<a href="@DateTime.()>Hello world</a>
```

Comment:
This is an error? I'd think this would be allowed and the `.()` would be HTML; Razor bug?